source/proof_document_mock/lib/document_mock_client.rb

```diff
@@ -0,0 +1,29 @@
+require 'proofer'
+
+module IdentityIdpFunctions
+  class DocumentMockClient < Proofer::Base
+    vendor_name 'DocumentMock'
+
+    required_attributes :encryption_key, :front_image_iv, :back_image_iv, :selfie_image_iv
+    required_attributes :front_image_url, :back_image_url, :selfie_image_url
+    required_attributes :liveness_checking_enabled
+
+    stage :document
+
+    UNVERIFIABLE_PHONE_NUMBER = '7035555555'.freeze
+    PROOFER_TIMEOUT_PHONE_NUMBER = '7035555888'.freeze
+    FAILED_TO_CONTACT_PHONE_NUMBER = '7035555999'.freeze
+
+    proof do |applicant, result|
+      plain_phone = applicant[:phone].gsub(/\D/, '').gsub(/\A1/, '')
+      if plain_phone == UNVERIFIABLE_PHONE_NUMBER
+        result.add_error(:phone, 'The phone number could not be verified.')
+      elsif plain_phone == FAILED_TO_CONTACT_PHONE_NUMBER
+        raise 'Failed to contact proofing vendor'
+      elsif plain_phone == PROOFER_TIMEOUT_PHONE_NUMBER
+        raise Proofer::TimeoutError, 'document mock timeout'
```

Comment:
On second look, I don't think we need this mock proofer class at all. The proof document step doesn't get `phone` passed in to it or anything, just images. So we'd need to YAML parse the input, or something?

This PR has been open long enough and I just want to get it merged to keep moving, but I think that the mock proofer should:
- Attempt to download & decrypt images (even if it doesn't process them, it lets us know the pipes work)
- Possibly parse the images as YAML to replicate current testing behavior